### PR TITLE
Allow plugin download to use an insecure repository

### PR DESCRIPTION
**Pull request by akram**
Allow plugin download to use an insecure repository (signed by unknown CA or self signed certificate)

### CODE REVIEW

**Comment by akram** [comment]
/assign @bparees 
@bparees can you PTAL ?
I tried to avoid introducing a security breach by using local variable.

**Comment by bparees** [comment]
/lgtm

but we're in a bit of holding pattern until our CI is back up which should be soon.

/hold

**Comment by gabemontero** [comment]
The install plugins command failed during the build:

```
[91m/usr/local/bin/install-plugins.sh: line 228: syntax error in conditional expression: unexpected token `;'
```

You change will need to be adjusted @akram

File: 2/contrib/jenkins/install-plugins.sh [hunk 1]
```diff
@@ -225,8 +225,13 @@ function doDownload() {
         url="$JENKINS_UC_DOWNLOAD/plugins/$plugin/$version/${plugin}.hpi"
     fi
 
+    if [[ "$JENKINS_UC_INSECURE" = true || "$JENKINS_UC_INSECURE" = 1]]; then
```
Comment (bparees) [inline 2/contrib/jenkins/install-plugins.sh]: i think it's a missing space between the `=1` and the `]]`  ?

**Comment by akram** [comment]
Due to deletion of my original branch and repo I recreated another pull request with the latest repo.

**Pull request closed without merging**